scenes/menus/title/main_menu/main_menu.gd

```diff
@@ -198,27 +198,8 @@ func _cycle_through(direction: int) -> void:
 
 
 func _manage_sizes(scale) -> void:
-	story.scale = Vector2.ONE * scale
-	settings.scale = Vector2.ONE * scale
-	extra.scale = Vector2.ONE * scale
-	ld.scale = Vector2.ONE * scale
-	selector_story.scale = Vector2.ONE * scale
-	selector_settings.scale = Vector2.ONE * scale
-	selector_extra.scale = Vector2.ONE * scale
-	selector_ld.scale = Vector2.ONE * scale
-	
-	icon.scale = Vector2.ONE * scale
-	border.rect_scale = Vector2.ONE * scale
-	border.margin_right = (OS.window_size.x + 1) / scale
-	border.margin_bottom = OS.window_size.y / scale
-	
-	options_menu.margin_top = 26 * scale
-	options_menu.margin_bottom = -4 * scale
-	options_menu.margin_left = 4 * scale
-	options_menu.margin_right = -4 * scale
-	
-	back_button.rect_scale = Vector2.ONE * scale
-	back_button.rect_pivot_offset.x = back_button.rect_size.x
+	rect_scale = Vector2.ONE * scale
```

Comment:
Why Vector2.ONE * scale?

Reply:
Because Vector2(scale, scale) is inconvenient, imo.

Reply:
Oops, I wasn't aware scale was a number, I thought it was a vector2, nvm then